src/bitmessagekivy/tests/test_myaddress_screen.py

```diff
@@ -0,0 +1,88 @@
+    @ordered
+    def test_show_Qrcode(self):
+        """Show the Qr code of selected address"""
+        # Checking current screen
+        self.assertExists("//ScreenManager[@current=\"myaddress\"]", timeout=3)
+        # Click on Address to open popup
+        self.cli.wait_click('//MDList[0]/CustomTwoLineAvatarIconListItem[0]', timeout=2)
+        # Check the Popup is opened
+        self.assertExists('//MyaddDetailPopup//MDLabel[@text=\"Show QR code\"]', timeout=2)
+        # Cick on 'Show QR code' button to view QR Code
+        self.cli.wait_click('//MyaddDetailPopup//MDLabel[@text=\"Show QR code\"]')
+        # Check Current screen is QR Code screen
+        self.assertExists("//ScreenManager[@current=\"showqrcode\"]", timeout=2)
+        # Click on BACK button
+        self.cli.wait_click('//MDToolbar//MDActionTopAppBarButton[@icon=\"arrow-left\"]', timeout=2)
+        # Checking current screen(My Address) after BACK press
+        self.assertExists("//ScreenManager[@current=\"myaddress\"]", timeout=3)
+
+    @skip_screen_checks
+    @ordered
+    def test_send_message_from(self):
+        """Send Message From send Button"""
+        # This is for checking Current screen
+        self.assert_wait_no_except('//ScreenManager[@current]', timeout=5, value='myaddress')
+        # Click on Address to open popup
+        self.cli.wait_click('//MDList[0]/CustomTwoLineAvatarIconListItem[0]', timeout=2)
+        # Checking Popup Opened
+        self.assertExists('//MyaddDetailPopup//MDLabel[@text=\"Send message from\"]', timeout=2)
+        # Click on Send Message Button to redirect Create Screen
+        self.cli.wait_click('//MyaddDetailPopup//MDRaisedButton[0]/MDLabel[0]', timeout=2)
+        # Checking Current screen(Create)
+        self.assertExists("//ScreenManager[@current=\"create\"]", timeout=2)
+        # Entering Receiver Address
+        self.cli.setattr(
+            '//DropDownWidget/ScrollView[0]//MyTextInput[0]', "text", data[1])
+        # Checking Receiver Address filled or not
+        self.assertNotEqual('//DropDownWidget//MyTextInput[0]', '')
+        # ADD SUBJECT
+        self.cli.setattr('//DropDownWidget/ScrollView[0]//MyMDTextField[0]', 'text', 'Hey this is Demo Subject')
+        # Checking Subject Field is Entered
+        self.assertNotEqual('//DropDownWidget/ScrollView[0]//MyMDTextField[0]', '')
+        # ADD MESSAGE BODY
+        self.cli.setattr(
+            '//DropDownWidget/ScrollView[0]//ScrollView[0]/MDTextField[0]', 'text',
+            'Hey,i am sending message directly from MyAddress book')
+        # Checking Message body is Entered
+        self.assertNotEqual('//DropDownWidget/ScrollView[0]//ScrollView[0]/MDTextField[@text]', '')
+        # Click on Send Icon
+        self.cli.wait_click('//MDActionTopAppBarButton[@icon=\"send\"]', timeout=3)
+        # Checking Current screen after Sending a message
+        self.assert_wait_no_except('//ScreenManager[@current]', timeout=5, value='inbox')
```

Comment:
we're not checking if message was queued for sending or if it shows up in sent folder